Extension auto-loads successfully:

```
❮ cd singleuser && docker compose run -e 'DJLABHUB_REPO=' --build -it singleuser -- python3 -m ipykernel --debug 
[+] Building 0.3s (11/11) FINISHED                                                                                             docker:default
 => [singleuser internal] load build definition from Dockerfile                                                                          0.0s
 => => transferring dockerfile: 1.45kB                                                                                                   0.0s
 => [singleuser internal] load metadata for quay.io/jupyter/minimal-notebook:hub-4.0.2                                                   0.0s
 => [singleuser internal] load .dockerignore                                                                                             0.0s
 => => transferring context: 2B                                                                                                          0.0s
 => [singleuser 1/6] FROM quay.io/jupyter/minimal-notebook:hub-4.0.2                                                                     0.0s
 => [singleuser internal] load build context                                                                                             0.0s
 => => transferring context: 6.15kB                                                                                                      0.0s
 => CACHED [singleuser 2/6] RUN if [ "$(python -c 'import sys; print(".".join(map(str, sys.version_info[:2])))')" != "3.11" ]; then      0.0s
 => CACHED [singleuser 3/6] COPY ./config /tmp/config                                                                                    0.0s
 => CACHED [singleuser 4/6] COPY ./ipython-datajoint-creds-updater /tmp/ipython-datajoint-creds-updater                                  0.0s
 => CACHED [singleuser 5/6] RUN     bash /tmp/config/apt_install.sh     && cp /tmp/config/before_start_hook.sh /usr/local/bin/before-no  0.0s
 => CACHED [singleuser 6/6] RUN     [ -d "/home/jovyan/work" ] && rm -r /home/jovyan/work     && pip install /tmp/ipython-datajoint-cre  0.0s
 => [singleuser] exporting to image                                                                                                      0.0s
 => => exporting layers                                                                                                                  0.0s
 => => writing image sha256:dcb9c47c712f379b1098b3838198c403806aea88ada58d5c599b4bf6a0efb3ec                                             0.0s
 => => naming to docker.io/datajoint/djlabhub:singleuser-4.0.2-py3.11                                                                    0.0s
Entered start.sh with args: -- python3 -m ipykernel --debug
Running hooks in: /usr/local/bin/start-notebook.d as uid: 1000 gid: 100
Done running hooks in: /usr/local/bin/start-notebook.d
Running hooks in: /usr/local/bin/before-notebook.d as uid: 1000 gid: 100
Sourcing shell script: /usr/local/bin/before-notebook.d/10activate-conda-env.sh
Sourcing shell script: /usr/local/bin/before-notebook.d/before_start_hook.sh
INFO::Datajoint Startup Hook
INFO::Changing Markdown Preview to preview as default
Done running hooks in: /usr/local/bin/before-notebook.d
Executing the command: -- python3 -m ipykernel --debug
0.00s - Debugger warning: It seems that frozen modules are being used, which may
0.00s - make the debugger miss breakpoints. Please pass -Xfrozen_modules=off
0.00s - to python to disable frozen modules.
0.00s - Note: Debugging will proceed. Set PYDEVD_DISABLE_FILE_VALIDATION=1 to disable this validation.
[IPKernelApp] Created profile dir: '/home/jovyan/.ipython/profile_default'
[IPKernelApp] Searching path ['/home/jovyan/.ipython/profile_default', '/opt/conda/etc/ipython', '/usr/local/etc/ipython', '/etc/ipython'] for config files
[IPKernelApp] Attempting to load config file: ipython_config.py
[IPKernelApp] Looking for ipython_config in /etc/ipython
[IPKernelApp] Looking for ipython_config in /usr/local/etc/ipython
[IPKernelApp] Looking for ipython_config in /opt/conda/etc/ipython
[IPKernelApp] Looking for ipython_config in /home/jovyan/.ipython/profile_default
[IPKernelApp] Attempting to load config file: ipython_kernel_config.py
[IPKernelApp] Looking for ipython_kernel_config in /etc/ipython
[IPKernelApp] Loaded config file: /etc/ipython/ipython_kernel_config.py
[IPKernelApp] Looking for ipython_kernel_config in /usr/local/etc/ipython
[IPKernelApp] Looking for ipython_kernel_config in /opt/conda/etc/ipython
[IPKernelApp] Looking for ipython_kernel_config in /home/jovyan/.ipython/profile_default
[IPKernelApp] Connection file not found: kernel-7.json
[IPKernelApp] Starting the kernel at pid: 7
[IPKernelApp] shell ROUTER Channel on port: 43941
[IPKernelApp] stdin ROUTER Channel on port: 46441
[IPKernelApp] control ROUTER Channel on port: 35855
[IPKernelApp] iopub PUB Channel on port: 42597
[IPKernelApp] Heartbeat REP Channel on port: 59133
[IPKernelApp] Writing connection file: /home/jovyan/.local/share/jupyter/runtime/kernel-7.json
[IPKernelApp] To connect another client to this kernel, use:
[IPKernelApp]     --existing kernel-7.json
NOTE: When using the `ipython kernel` entry point, Ctrl-C will not work.

To exit, you will have to explicitly quit this process, by either sending
"quit" from a client, or using Ctrl-\ in UNIX-like environments.

To read more about this, see https://github.com/ipython/ipython/issues/2049


To connect another client to this kernel, use:
    --existing kernel-7.json
[IPKernelApp] Seeing logger to stderr, rerouting to raw filedescriptor.
[IPKernelApp] Loading IPython extensions...
[IPKernelApp] Loading IPython extension: storemagic
[IPKernelApp] Loading IPython extension: ipython_datajoint_creds_updater.extension
[IPKernelApp] Using IPython logger.
[IPKernelApp] ERROR | Plugin failed to load JHubConfig: 3 validation errors for JHubConfig
jupyterhub_api_url
  Field required [type=missing, input_value={}, input_type=dict]
    For further information visit https://errors.pydantic.dev/2.8/v/missing
jupyterhub_api_token
  Field required [type=missing, input_value={}, input_type=dict]
    For further information visit https://errors.pydantic.dev/2.8/v/missing
jupyterhub_user
  Field required [type=missing, input_value={}, input_type=dict]
    For further information visit https://errors.pydantic.dev/2.8/v/missing
[IPKernelApp] WARNING | Error in loading extension: ipython_datajoint_creds_updater.extension
Check your config files in /home/jovyan/.ipython/profile_default
Traceback (most recent call last):
  File "/opt/conda/lib/python3.11/site-packages/ipython_datajoint_creds_updater/extension.py", line 122, in load_ipython_extension
    JHubConfig()
  File "/opt/conda/lib/python3.11/site-packages/pydantic_settings/main.py", line 140, in __init__
    super().__init__(
  File "/opt/conda/lib/python3.11/site-packages/pydantic/main.py", line 193, in __init__
    self.__pydantic_validator__.validate_python(data, self_instance=self)
pydantic_core._pydantic_core.ValidationError: 3 validation errors for JHubConfig
jupyterhub_api_url
  Field required [type=missing, input_value={}, input_type=dict]
    For further information visit https://errors.pydantic.dev/2.8/v/missing
jupyterhub_api_token
  Field required [type=missing, input_value={}, input_type=dict]
    For further information visit https://errors.pydantic.dev/2.8/v/missing
jupyterhub_user
  Field required [type=missing, input_value={}, input_type=dict]
    For further information visit https://errors.pydantic.dev/2.8/v/missing

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/opt/conda/lib/python3.11/site-packages/IPython/core/shellapp.py", line 282, in init_extensions
    self.shell.extension_manager.load_extension(ext)
  File "/opt/conda/lib/python3.11/site-packages/IPython/core/extensions.py", line 62, in load_extension
    return self._load_extension(module_str)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/conda/lib/python3.11/site-packages/IPython/core/extensions.py", line 79, in _load_extension
    if self._call_load_ipython_extension(mod):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/conda/lib/python3.11/site-packages/IPython/core/extensions.py", line 129, in _call_load_ipython_extension
    mod.load_ipython_extension(self.shell)
  File "/opt/conda/lib/python3.11/site-packages/ipython_datajoint_creds_updater/extension.py", line 125, in load_ipython_extension
    raise Exception("Plugin failed to load JHubConfig") from e
Exception: Plugin failed to load JHubConfig
```

--- a/singleuser/Dockerfile
+++ b/singleuser/Dockerfile
@@ -29,4 +29,4 @@ RUN \
     # remove default work directory
     [ -d "/home/jovyan/work" ] && rm -r /home/jovyan/work \
     # Install dependencies: pip
-    && pip install /tmp/ipython-datajoint-creds-updater -r /tmp/config/pip_requirements.txt jupyter_contrib_nbextensions
+    && pip install /tmp/ipython-datajoint-creds-updater -r /tmp/config/pip_requirements.txt

